isis/tests/FunctionalTestsGetsn.cpp

```diff
@@ -22,18 +22,18 @@ static QString APP_XML = FileName("$ISISROOT/bin/xml/getsn.xml").expanded();
 
 // check for all correct outputs
 TEST_F(DefaultCube, FunctionalTestGetsnAllTrue) {
+  QString d_APP_XML = FileName("$ISISROOT/bin/xml/getsn.xml").expanded();
```

Comment:
If static variables are an issue in test cases, we should get rid of the static variable and switch all of the test cases over to using this. Another option would be to sub-class the DefaultCube and put the app xml into the fixture.